avm/src/main.rs

```diff
@@ -20,8 +21,10 @@ pub enum Commands {
     },
     #[clap(about = "Install a version of Anchor")]
     Install {
-        #[clap(value_parser = parse_version)]
-        version: Version,
+        /// Anchor version or commit
+        version: String,
+        #[clap(short, long, required = false)]
+        is_commit: bool,
```

Comment:
>  Single arg trying ver then commit, less clunky than the extra flag

I'm guessing this is not implemented yet. Don't think `is_commit` flag is necessary, can just decide whether it's a version or a commit based on the input.

Reply:
Forgot to push, pushed and back to single arg

Trying to install an odd commit
```
target/debug/avm install dcafb789e1635b8f90e8fd3badd0f9a015d204d42ds2`
error: invalid value 'dcafb789e1635b8f90e8fd3badd0f9a015d204d42ds2' for '<VERSION_OR_COMMIT>': Not a valid version or commit: unexpected character 'd' while parsing major version number, Error checking commit dcafb789e1635b8f90e8fd3badd0f9a015d204d42ds2: {"message":"No commit found for SHA: dcafb789e1635b8f90e8fd3badd0f9a015d204d42ds2","documentation_url":"https://docs.github.com/rest/commits/commits#get-a-commit"}
```